kubernetes/cache/proxy_status.go

```diff
@@ -38,7 +38,7 @@ func (c *kialiCacheImpl) pollIstiodForProxyStatus(ctx context.Context) {
 					)
 
 					interval := c.tokenNamespaceDuration / 2
-					retryErr := wait.PollImmediateUntilWithContext(ctx, interval, func(ctx context.Context) (bool, error) {
+					retryErr := wait.PollUntilContextCancel(ctx, interval, false, func(ctx context.Context) (bool, error) {
```

Comment:
```suggestion
					retryErr := wait.PollUntilContextCancel(ctx, interval, true, func(ctx context.Context) (bool, error) {
```

nit: looks like this should pass `true` to keep the same behavior as `immediate` where the polling function starts immediately.